README.md

```diff
@@ -82,6 +82,28 @@ export class SharedModule {
 }
 ```
 
+__If you use AoT compiling or Ionic 2 you MUST to use a exported function for `useFactory`:__
```

Comment:
Could you change it to: "If you use a custom `TranslateLoader` and use AoT compiling or Ionic 2" please ? Thanks